### ISSUE TITLE
Add aria accessibility for screen readers

### ISSUE THREAD

**Issue by chamandeep** [wenzhixin/multiple-select#139]
This plugin could be made more accessible to blind users who are using screen reader software by adding wai-aria. For instance, when user is filtering through a large list, as they type, they should be able to hear how many results have been found. This is possible through aria.

**Comment by wenzhixin** [comment]
You can use `onFilter` to solve your problem.
